bemani/backend/jubeat/clan.py

```diff
@@ -1105,6 +1105,7 @@ def format_scores(self, userid: UserID, profile: ValidatedDict, scores: List[Sco
 
         music = ValidatedDict()
         for score in scores:
+            chart = score.chart if score.chart < 3 else score.chart - 3
```

Comment:
Again, should be a specific game_to_db_chart/db_to_game_chart lookup, even if its a lot more code. That makes it searchable and prevents coupling of game features to math that works "for now".